modules/claims_api/spec/services/failed_submissions_messenger_spec.rb

```diff
@@ -11,25 +11,41 @@
   let(:itf) { ['24252627'] }
   let(:ews) { %w[32333435 36373839] }
   let(:from) { '2022-01-01' }
-  let(:to) { '2022-01-31' }
+  let(:to) { '2022-01-01' }
```

Comment:
This date range was just an oversight when I originally set this up.  It runs on the hour not month so just fixed it. Doesn't really matter but felt. like a typo since 01 -31 doesn't really make sense in the context of this job.